.github/workflows/verification.yml

```diff
@@ -0,0 +1,19 @@
+name: Verification
+
+on:
+  pull_request:
+    branches: [master]
+  push:
+    branches: [master]
```

Comment:
Are you sure it is running on the all the commits that we have?

what is the difference with this?

`on: [ push, pull_request ]`